x-pack/plugins/observability_solution/slo/server/services/get_diagnosis.ts

```diff
@@ -8,14 +8,36 @@
 import { ElasticsearchClient } from '@kbn/core-elasticsearch-server';
 import { LicensingApiRequestHandlerContext } from '@kbn/licensing-plugin/server';
 
+export const MINIMUM_INDEX_PRIVILEGE_SET_EDITOR = [
+  'write',
+  'read',
+  'view_index_metadata',
+  'manage',
+  'auto_configure',
+];
+export const TOTAL_INDEX_PRIVILEGE_SET_EDITOR = [
+  'write',
+  'read',
+  'read_cross_cluster',
+  'view_index_metadata',
+  'manage',
+  'auto_configure',
+];
+export const MINIMUM_INDEX_PRIVILEGE_SET_VIEWER = ['read'];
+export const TOTAL_INDEX_PRIVILEGE_SET_VIEWER = ['read', 'read_cross_cluster'];
+
 export async function getGlobalDiagnosis(
   esClient: ElasticsearchClient,
   licensing: LicensingApiRequestHandlerContext
 ) {
   const licenseInfo = licensing.license.toJSON();
   const userWritePrivileges = await esClient.security.hasPrivileges({
-    cluster: ['manage_transform', 'manage_ingest_pipelines'],
-    index: [{ names: '.slo-*', privileges: ['all'] }],
+    index: [
+      {
+        names: '.slo-*',
+        privileges: ['write', 'read', 'view_index_metadata', 'manage', 'auto_configure'],
```

Comment:
@dominiqueclarke Could you use here `MINIMUM_INDEX_PRIVILEGE_SET_EDITOR` you defined above?